### ISSUE TITLE
`bundle: command not found` when setting up the simulator following the README

### ISSUE THREAD

**Issue by erinaldi** [crest-cassia/oacis_docker#88]
# Summary

Following the instructions in the main README file I have started the `oacis_jupyter` container with
```shell
docker run --name my_oacis -p 127.0.0.1:3000:3000 -p 127.0.0.1:8888:8888 -dt oacis/oacis_jupyter
```
It started as expected and I could exec into it. I could also reach the main dashboard as can be seen in the screenshot:
<img width="1181" alt="Screen Shot 2022-01-19 at 16 22 36" src="https://user-images.githubusercontent.com/5943556/150083302-87e5d32b-9547-4f57-9282-4e6957090f7b.png">
 
When I try to setup the simulator using the script provided and the command
```shell
docker exec -t -u oacis my_oacis bash /home/oacis/setup_ns_model.sh
```
I get an error about `bundle` being missing.

# Report

Here is the full output of the simulator setup command
```
+ git clone https://github.com/yohm/nagel_schreckenberg_model.git
Cloning into 'nagel_schreckenberg_model'...
remote: Enumerating objects: 66, done.
remote: Counting objects: 100% (3/3), done.
remote: Compressing objects: 100% (3/3), done.
remote: Total 66 (delta 0), reused 2 (delta 0), pack-reused 63
Unpacking objects: 100% (66/66), done.
+ cd nagel_schreckenberg_model
+ bundle install
/home/oacis/setup_ns_model.sh: line 5: bundle: command not found
```

**Comment by erinaldi** [comment]
I think `bundle` only gets installed in the `$OACIS_ROOT/bin` folder, so you have to add it to the `PATH` inside the container

**Comment by yohm** [comment]
Thank you for reporting this issue. The command on README was wrong. It should be

```
docker exec -t -u oacis my_oacis bash -l /home/oacis/setup_ns_model.sh
```

to launch bash as a login shell and load `.bash_profile` to set up environment variables.

**Comment by yohm** [comment]
The readme is fixed by #89.

**Comment by erinaldi** [comment]
Thank you.
The new command works and here is the output:
```shell
(base) physics:Projects$ docker exec -t -u oacis my_oacis bash -l /home/oacis/setup_ns_model.sh
+ git clone https://github.com/yohm/nagel_schreckenberg_model.git
Cloning into 'nagel_schreckenberg_model'...
remote: Enumerating objects: 66, done.
remote: Counting objects: 100% (3/3), done.
remote: Compressing objects: 100% (3/3), done.
remote: Total 66 (delta 0), reused 2 (delta 0), pack-reused 63
Unpacking objects: 100% (66/66), done.
+ cd nagel_schreckenberg_model
+ bundle install
Fetching gem metadata from https://rubygems.org/.
Resolving dependencies...
Using bundler 2.2.16
Using chunky_png 1.4.0
Bundle complete! 1 Gemfile dependency, 2 gems now installed.
Use `bundle info [gemname]` to see where a bundled gem is installed.
+ /home/oacis/oacis/bin/oacis_ruby register_on_oacis.rb
```
<img width="1164" alt="Screen Shot 2022-01-20 at 9 33 31" src="https://user-images.githubusercontent.com/5943556/150240706-03a52fdf-f502-498f-89a3-ed335c097f05.png">

After this, the simulator shows up on the dashboard.
